src/Component/AutoMapper/AutoMapper.php

```diff
@@ -204,6 +206,12 @@ public static function create(
 
         if ($private) {
             $flags |= ReflectionExtractor::ALLOW_PROTECTED | ReflectionExtractor::ALLOW_PRIVATE;
+        } else {
+            trigger_deprecation('jane-php/automapper', '7.6.0', 'Setting `$private` to false is deprecated. This argument will be removed in Jane 8.0.');
+
+            if ($mapPrivateProperties) {
+                throw new \LogicException('Cannot set `$private` to false and `$mapPrivateProperties` to true at the same time.');
```

Comment:
not sure of this... it might create a BC break when creating automapper with `$private = false`
in the meantime, `$private = false` + `$mapPrivateProperties = true` cannot work... 

WDYT?

Reply:
Since we will have a major once that PR is merged, we don't need the deprecation layer, you can just remove `$private` and replace it with `$mapPrivateProperties` (but that will make some tests fail ?)

Reply:
perfect, I'll fix the failing tests!